artiq/coredevice/comm_kernel.py

```diff
@@ -701,6 +705,20 @@ def _process_async_error(self):
             logger.warning(f"{(', '.join(errors[:-1]) + ' and ') if len(errors) > 1 else ''}{errors[-1]} "
                            f"reported during kernel execution")
 
+    def config_channel_name(self, channel_number):
+        device_db = self.dmgr.get_device_db()
+        for device_name, value in device_db.items():
+            try:
+                device_channel_number = value["arguments"]["channel"]
+            except:
```

Comment:
I don't know how many times I have already repeated that I don't want indiscriminate ``except: pass`` clauses.